qualtran/surface_code/fifteen_to_one.py

```diff
@@ -378,6 +380,10 @@ def _build_factory(
     return factory
 
 
-FifteenToOne733 = FifteenToOne(7, 3, 3, reference='https://arxiv.org/abs/1905.06903')
+FifteenToOne733 = FifteenToOne(
+    7, 3, 3, reference=Reference(source='https://arxiv.org/abs/1905.06903')
```

Comment:
extreme nit: you can probably get rid of the `reference=` kwarg now since it's obvious from the (new) class name

Reply:
not really, there is another parameter before reference